gfauto/gfauto/download_cts_gf_tests.py

```diff
@@ -99,17 +99,26 @@ def download_cts_graphicsfuzz_tests(  # pylint: disable=too-many-locals;
 
     subprocess_util.run(cmd, verbose=True, working_dir=work_dir)
 
-    shader_dir = util.copy_dir(
+    return util.copy_dir(
         cts_out
         / "external"
         / "vulkancts"
         / "data"
         / "vulkan"
         / "amber"
         / "graphicsfuzz",
-        Path() / "graphicsfuzz",
+        output_shaders_dir,
     )
 
+
```

Comment:
Am I right that this is something only available to Khronos group members?  Worth documenting that explicitly?

Reply:
(I see that this was already present and just moved in this PR.)

Reply:
Done